mobilizon_bots/storage/query.py

```diff
@@ -93,6 +102,21 @@ async def create_publisher(name: str, account_ref: Optional[str] = None) -> None
     await Publisher.create(name=name, account_ref=account_ref)
 
 
-async def save_publication_report(publication_report: PublisherCoordinatorReport):
+@atomic(CONNECTION_NAME)
+async def save_publication_report(
+    publication_report: PublisherCoordinatorReport, event: MobilizonEvent
+):
+    publications: dict[str, Publication] = {
```

Comment:
this could behave weirdly if at some point we have multiple publications for the same publisher.